omod/src/main/compass/sass/allergy.scss

```diff
@@ -7,6 +7,11 @@
     }
   }
 
+  #types {
+  margin: 10px 0px;
```

Comment:
Do you mind explaining how this is related to the disabling and enabling of the textbox?

Reply:
When you look in the allergy.gsp line 61
> <div id="allergens">
                <div id="types" class="button-group horizontal">
                    <% allergenTypes.each { category ->  if (category.name()!="ENVIRONMENT") { %>
                    <label class="button small" ng-model="allergenType" btn-radio="'${ui.format(category)}'" ng-class="{ confirm: allergenType == '${ui.format(category)}' }">
                        ${ category }
                    </label>
                    <% } } %> </div>

margin... spaces it since there is a dropdown of the other (textbox) of the allergen which helps in not expanding the allergen list when there is a dropdown